devito/ops/transformer.py

```diff
@@ -37,7 +42,44 @@ def opsit(trees, count):
         parameters
     )
 
-    return ops_kernel
+    stencil_arrays_initializations = itertools.chain(*[
+        to_ops_stencil(p, node_factory.ops_args_accesses[p])
+        for p in parameters if isinstance(p, OpsAccessible)
+    ])
+
+    pre_time_loop = stencil_arrays_initializations
+
+    return pre_time_loop, ops_kernel
+
+
+def to_ops_stencil(param, accesses):
+    dims = len(accesses[0])
+    pts = len(accesses)
+    stencil_name = "s%sd_%s_%spt" % (dims, param.name, pts)
+
+    stencil_array = SymbolicArray(
```

Comment:
remind me what is this for, that is, what kind of C is it supposed to generate?

Reply:
We want to generate `int array[size] = { x, x};`
If we use a normal Array we get no type (we'd get `array[size] = { x, x };`)

Reply:
what, really? do you have a (minimal) reproducer by any chance? I'd be happy to take a look. This could be relevant for several other things

Reply:
If you take this branch and change this line to use `Array` you can observe it on any example (I've been using `example_diffusion`)